content/ADR-145-pick-realm-v2.md

```diff
@@ -3,49 +3,69 @@ layout: adr
 adr: 145
 title: Pick Realm Algorithm V2
 date: 2022-11-28
-status: Draft
+status: Living
```

Comment:
We only make ADR living when they are running in production

Reply:
Got it, moving this back to Draft, thanks!

Reply:
Done here -> https://github.com/decentraland/adr/pull/220/commits/3d60c885159e2dd9a286a48b3865cdfa3fdce0cf